docs/runtimes.md

```diff
@@ -331,6 +332,40 @@ You can deploy them using the command:
 kubeless function deploy fibonacci --from-file fibonacci.cs --handler module.handler --dependencies fibonacci.csproj --runtime dotnetcore2.0
 ```
 
+### Ballerina
+
+#### Example
+
+```ballerina
+import kubeless;
+import ballerina/io;
+
+public function foo(kubeless:Event event, kubeless:Context context) returns (string|error) {
+    io:println(event);
+    io:println(context);
+    return "Hello Ballerina";
+}
+
+```
+
+#### Description
+
+Ballerina functions should import the package `kubeless`. This [package](../docker/runtime/ballerina/kubeless/kubeless.bal) contains two types `Event` and `Context`. 
+
+When using the Ballerina runtime, it is possible to provide the `ballerina.conf` file. The values in conf file is available for the function. 
```

Comment:
I think we shouldn't use the flag `--dependencies` for other thing than installing external dependencies. Since ballerina doesn't require a different file to do so I think it's better to just ignore this flag. If people wants to use a configuration file they can bundle their files in a zip that will be uncompressed.